how-to-guides/hello-neptune/notebooks/hello_neptune.ipynb

```diff
@@ -332,19 +347,20 @@
    "source": [
     "## Explore the run in the Neptune app\n",
     "\n",
-    "* Uploaded source files are present in the **Source code** dashboard\n",
-    "* The graphs of the `accuracy` and `loss` fields are displayed in the **Charts** dashboard\n",
-    "* You can view your system hardware consumption in the **Monitoring** dashboard\n",
-    "* The images logged are in the **Images** dashboard"
+    "* View the uploaded source files in the **Source code** dashboard\n",
```

Comment:
Is this applicable? For me the dashboard is empty.

Reply:
The init call is quite far away by this point. Let's mention that the **Source code** tab will be empty if you ran the notebook in Colab. To see source files here, you need to run the notebook locally and use the `source_files` argument to specify the files.